drivers/arm/i2c_master.c

```diff
@@ -58,28 +58,28 @@ void i2c_init(void)
 }
 
 // This is usually not needed
```

Comment:
Lets get rid of this comment, it's definitely wrong.

Reply:
Removed